### DISCUSSION TITLE
How to set the correct path in a Finder window in App Layouts

### DISCUSSION THREAD

**Discussion by pekka** [rxhanson/RectanglePro-Community#275] [General]
This is a Rectangle Pro question, hope that's ok here...

I would like to use the App Layouts feature to quickly switch between workspaces. So, eg for video editing, Premiere on the main screen, a finder window with my video projects and a browser with the music library on the secondary screen.

This works great, but I can't seem to save the Finder window with the correct path, nor can I add it afterward - I can only select the app and not pass any parameters. Same for the browser.

Is there a way to achieve what I'm looking for? Perhaps using a script I'd be calling instead?

**Comment by rxhanson** [comment]
Thanks for the feedback. There are some macOS limitations that get in the way from doing this in an elegant way out of the box. There would likely be some delicate timing delays set up to get an app window open and direct it to the right place. I'll give this some thought for a future update, though.
